### PR TITLE
Concf 477 fix background on breakpoints

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
ping @rogatty @Warkot

**Comment by rogatty** [comment]
I'm no compiler but LGTM after a quick look.

**Comment by Warkot** [comment]
Please run
```
scss-lint --config ../guidelines/SCSS/.scss-lint.yml skins/oasis/css/core/breakpoints-background.scss
```
(Or wherever you have cloned the [guidlines repo](https://github.com/Wikia/guidelines/). Did you clone it? I'm sure you did! :smiling_imp:). I tried to fix most of the issues, there were some severe ones (you can run the SCSS lint on the previous commit to see them → e.g. merging some rules like `.background-image-gradient`). I hope I didn't break anything (please check, I just tried to follow SCSS's logic). There are still few more issues, but I didn't want to touch them without the full context. Please take care of them if possible.

File: skins/oasis/css/core/breakpoints-background.scss [hunk 1]
```diff
@@ -1,93 +1,75 @@
...
+			&::before {
+				@include calc('left', '50% - #{$gradient-offset}');
 			}
 		}
 	}
 }
 
-
-body.skin-oasis {
+body {
```
Comment (vforge) [inline skins/oasis/css/core/breakpoints-background.scss]: Are you sure we want to change background for non-oasis skin?

File: skins/oasis/css/core/breakpoints-background.scss [hunk 2]
```diff
@@ -1,93 +1,75 @@
...
+$background-align: if($background-width > $width-outside, center, left);
+$gradient-offset: $legacy-background-shift + $gradient-width;
 
-	background-color: $color-body-middle-0;
-	background-image: -webkit-linear-gradient(#{$side}, $color-body-middle-0 0%, $color-body-middle 100%);
-	background-image: linear-gradient(to #{$other-side}, $color-body-middle-0 0%, $color-body-middle 100%);
-	margin-#{$side}: $margin - $width-gradient;
-	#{$side}: 50%;
+@mixin breakpoint-middle-gradient($side) {
+	background: rgba($color-body-middle, 0) linear-gradient(to #{$side}, rgba($color-body-middle, 0) 0%, $color-body-middle 100%);
```
Comment (vforge) [inline skins/oasis/css/core/breakpoints-background.scss]: Is missing -webkit prefix intended?

File: skins/oasis/css/core/breakpoints-background.scss [hunk 3]
```diff
@@ -1,93 +1,75 @@
...
+			width: $background-width-half;
 		}
 
 		&:after {
-			background-position: $background-position-after 0;
-			/* @noflip */
-			right: calc(50% + #{$background-offset});
+			background-position: -$background-width-half 0;
+			margin-right: $background-margin;
+			right: 50%;
```
Comment (vforge) [inline skins/oasis/css/core/breakpoints-background.scss]: You need to keep `/* @noflip */` for Janus

File: skins/oasis/css/core/breakpoints-background.scss (hunk 2 above)
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: yes. http://caniuse.com/#search=linear-gradient

Comment (vforge) [inline skins/oasis/css/core/breakpoints-background.scss]: :heavy_check_mark:

**Comment by vforge** [comment]
LGTM :+1:

**Comment by Warkot** [comment]
LGTM

**Pull request merged**